ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -197,10 +197,12 @@ public HelixClusterManager(ClusterMapConfig clusterMapConfig, String instanceNam
   private HelixManager initializeHelixManagerAndPropertyStoreInLocalDC(Map<String, DcZkInfo> dataCenterToZkAddress,
       String instanceName, HelixFactory helixFactory) throws Exception {
     DcZkInfo dcZkInfo = dataCenterToZkAddress.get(clusterMapConfig.clusterMapDatacenterName);
-    String zkConnectStr = dcZkInfo.getZkConnectStr();
     if (dcZkInfo.getReplicaType() == ReplicaType.CLOUD_BACKED) {
       return null;
     }
+    // For now, the first ZK endpoint (if there are more than one endpoints) will be adopted by default. We can update
+    // this if we really need to initialize multiple HelixManagers in local dc.
```

Comment:
I don't think we need to support multiple helix instances here, maybe we should change the comments.

Reply:
hmm, I don't see any use cases for now, will remove the comment.